src/integrated/soil_canopy_model.jl

```diff
@@ -420,7 +420,8 @@ function soil_boundary_fluxes!(
     p.soil.turbulent_fluxes .= turbulent_fluxes(bc.atmos, soil, Y, p, t)
     Soil.Runoff.update_runoff!(p, bc.runoff, Y, t, soil)
     @. p.soil.top_bc.water =
-        p.soil.infiltration + p.soil.turbulent_fluxes.vapor_flux
+        p.soil.infiltration +
+        p.soil.turbulent_fluxes.vapor_flux * (1 - p.soil.ice_frac)
```

Comment:
It might be helpful to add a comment here that we multiply the vapor flux by `1 - (p.soil.ice_frac)` to get the water component